index.bs

```diff
@@ -1911,14 +1911,16 @@ to indicate what [=credential sources=] are acceptable to it. The [=client platf
 matching the specified criteria, and guides the user to pick one that the script will be allowed to use. The user may choose to
 decline the entire interaction even if a [=credential source=] is present, for example to maintain privacy. If the user picks a
 [=credential source=], the user agent then uses
-[[#sctn-op-get-assertion]] to sign a [=[RP]=]-provided challenge and other collected data into an assertion, which is used as a
+[[#sctn-op-get-assertion]] to sign a [=[RP]=]-provided challenge and other collected data into an [=assertion=], which is used as a
 [=credential=].
 
-The {{CredentialsContainer/get()}} implementation [[!CREDENTIAL-MANAGEMENT-1]] calls
+The {{CredentialsContainer/get()|navigator.credentials.get()}} implementation [[!CREDENTIAL-MANAGEMENT-1]] calls
 <code>PublicKeyCredential.{{PublicKeyCredential/[[CollectFromCredentialStore]]()}}</code> to collect any [=credentials=] that
 should be available without [=user mediation=] (roughly, this specification's [=authorization gesture=]), and if it does not find
-exactly one of those, it then calls <code>PublicKeyCredential.{{PublicKeyCredential/[[DiscoverFromExternalSource]]()}}</code> to have
-the user select a [=credential source=].
+exactly one of those, it then calls <code>PublicKeyCredential.{{PublicKeyCredential/[[DiscoverFromExternalSource]]()}}</code> to have the user select a [=credential source=].
+
+In any case, the user agent SHOULD show some UI to the user to guide them in selecting and
+authorizing an authenticator with which to complete the operation. [=[RPS]=] can provide a hint that a prominent, modal UI <em>should not</em> be used for this process by setting <code>|options|.{{CredentialRequestOptions/mediation}}</code> to "conditional". This is known as a <dfn>Conditional UI Flow</dfn>. [=[RP]=] script SHOULD first check that <code>navigator.credentials.{{CredentialsContainer/conditionalMediationSupported}}</code> is [TRUE] in order to avoid the possiblity of causing a user-visible error to be returned if the user agent does not support [=conditional UI flow=].
```

Comment:
I just caught myself up on this proposal. On the whole I think this is a great idea. However I suggest we consider the following alternative to the naming of the boolean `conditionalMediationSupported` :

```
navigator.credentials.isConditionalMediationAvailable()
```

I suggest this because we already have the existing helper method `isUserVerifyingPlatformAuthenticatorAvailable()` that establishes an `is...Available()` naming convention for these kinds of helper methods. I'm also suggesting this becomes a **method** instead of a **boolean** because I believe it will eliminate confusion over whether a particular feature check in the WebAuthn API is a simple boolean, or something that needs to be invoked. If they're all invoked methods that return booleans then there's less mental overhead for RP developers working with this API.

Reply:
Thanks @MasterKale, looks like this feedback belongs also in https://github.com/w3c/webappsec-credential-management/pull/155 which is what would add that attribute.

Reply:
Thank you for the nudge, I went ahead and cross-posted. Honestly I feel like this suggestion is less compelling over there without something like `isUserVerifyingPlatformAuthenticatorAvailable()` in the credential management API (which there isn't)...

If conditional mediation goes through with `conditionalMediationSupported`, is there any value in trying to update `PublicKeyCredential` with a new static method `isConditionalMediationAvailable()` that returns the value of `navigator.credentials.conditionalMediationSupported`? 😛

Reply:
A drive-by thought: Is it certain that this kind of support will always be a single boolean? The function leaves open the possibility of arguments in the future, in case the level of support depends on certain properties (e.g. of the registration parameters) in a non-trivial way.

Reply:
ISTM @MasterKale makes good suggestions wrt naming and it being a method rather than a simple boolean.

Reply:
@emlun, @MasterKale (and other reviewers!): 
WRT @nsatragno's https://github.com/w3c/webauthn/pull/1576#discussion_r736620812 immediately above regarding  `isConditionalMediationAvailable()` being usable by devs as simply `PublicKeyCredential.isConditionalMediationAvailable()`, please also see the current state of w3c/webappsec-credential-management#155 -- We're feeling it is getting very close to being baked.